### PR TITLE
Features/vcf2cytosure install

### PR DESCRIPTION
**Pull request by northwestwitch**
Something like this?

### CODE REVIEW

File: lib/MIP/Recipes/Install/Vcf2cytosure.pm [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    say {$FILEHANDLE} q{## Move to vcf2cytosure directory};
+    gnu_cd(
+        {
+            directory_path => catfile( $temp_dir, q{vcf2cytosure} ),
+            FILEHANDLE     => $FILEHANDLE,
+        }
+    );
+    say {$FILEHANDLE} $NEWLINE;
+
+    ## Pip install the downloaded SVDB package
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Install/Vcf2cytosure.pm]: SVDB -> vcf2cytosure

**Comment by jemten** [comment]
Looks ace!

**Comment by northwestwitch** [comment]
Yeah, it's where I took inspiration from! :)

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**